ztp/resource-generator/README.md

```diff
@@ -0,0 +1,88 @@
+# README #
+
+This directory provides RAN site ArgoCD gitops pipeline container image definitions
+
+## What is this repository for? ##
+### Quick summary ###
+
+The idea is to accept RAN site manifests in a format defined by "SiteConfig" CRD, convert them to custom resources accepted by the assisted service and ACM, and apply them to the cluster running Assisted Service Operator and ACM hub.
```

Comment:
The plain is to use the same container to generate the acm-policies using PolicyGenTemplate CRD. 

Are you planing to update this readme later ? sounds like it follows the old directory structure where everything under the same argocd dir. 

I feel Its better to make this readme only for how to build and execute the image ? what do u think ?

Reply:
I hear you. And yet I think this or similar content should be defined somewhere. Maybe when we do the changes we discussed for policy generator, this directory will be bound to to a massive overhaul?